libethereum/EthereumHost.cpp

```diff
@@ -58,6 +58,9 @@ EthereumHost::EthereumHost(BlockChain const& _ch, OverlayDB const& _db, Transact
 	m_bq		(_bq),
 	m_networkId	(_networkId)
 {
+	// TODO: Composition would be better. Left like that to avoid initialization
+	//       issues as BlockChainSync accesses other EthereumHost members.
+	m_sync.reset(new BlockChainSync(*this));
```

Comment:
This is unique_ptr. If you want to replace missing make_unique I think this is bad idea. make_shared allocates more memory than needed.